benchmarks/linear_algebra/blas/level1/swap/swap.cpp

```diff
@@ -0,0 +1,66 @@
+/**
+  swap : swap 2 vectors content 
+
+
```

Comment:
Thanks! The code looks good to me, but I have one comment. The code does not respect the Tiramisu coding style. For example, the code should not have a new line when not necessary.

Reply:
fixed.